index.html

```diff
@@ -7,6 +7,7 @@
     <meta name="autor" content="Anton and Irina Bocharov">
     <meta name="vienport" content="width=device-width, initial-scale=1.0">
     <title>Моя первая страница</title>
+    <link rel="stylesheet" href="./styles/styles.css">
```

Comment:
Create new branch for the next task pls.

Reply:
ok